### PR TITLE
truncate branch name to match cloudflare

### PR DESCRIPTION
**Pull request by jharrell**
Resolves DA-325

### CODE REVIEW

File: .github/workflows/lost-pixel.yml
Comment (janpio) [inline .github/workflows/lost-pixel.yml:1]: Unrelated change to PR?

**Review by janpio — APPROVED** [review]

File: .github/workflows/lost-pixel.yml
Comment (jharrell) [reply .github/workflows/lost-pixel.yml:1]: [`3549b6b` (#6088)](https://github.com/prisma/docs/pull/6088/commits/3549b6b56689c3bca89f0d3b22a0fcee2ba35479)
